ts/components/WalletTabIcon.tsx

```diff
@@ -0,0 +1,84 @@
+import * as pot from "italia-ts-commons/lib/pot";
+import { Badge, View } from "native-base";
+import React from "react";
+import { Platform, StyleSheet, Text } from "react-native";
+import { connect } from "react-redux";
+import { getUnreadTransactionsSelector } from "../store/reducers/entities/readTransactions";
+import { GlobalState } from "../store/reducers/types";
+import { makeFontStyleObject } from "../theme/fonts";
+import variables from "../theme/variables";
+import IconFont from "./ui/IconFont";
+
+type OwnProps = {
+  color?: string;
+};
+const MAX_BADGE_VALUE = 99;
+const styles = StyleSheet.create({
+  textBadgeStyle: {
+    ...makeFontStyleObject(
+      Platform.select,
```

Comment:
Just pass only the first 2 parameters.

Reply:
Updated within 2cbc97e67760ab08a0e2930ae032f4ce4b76d3c5
I created a generic tab icon component to render the badge on the tab icon in a generalized way. Now it uses the native-base Text component (tab icon badge number was previously rendered by using the react-native component).